src/components/Modal/Modal.test.tsx

```diff
@@ -2,44 +2,46 @@ import { render, screen } from '@testing-library/react';
 import Modal from './Modal';
 import userEvent from '@testing-library/user-event';
 import { runCommonTests } from '../../utils/testUtils';
+import { ModalFromDrawer } from './Modal.stories';
 
 describe('Modal', () => {
   const onCloseMock = vi.fn();
 
   afterEach(() => {
     vi.clearAllMocks();
   });
-  runCommonTests((props) => <Modal {...props} isOpen appElementSelector="body"></Modal>, 'Modal');
+  runCommonTests((props) => <Modal {...props} isOpen></Modal>, 'Modal');
```

Comment:
Should probably be self-closing
```suggestion
  runCommonTests((props) => <Modal {...props} isOpen />, 'Modal');
```